internal/pkg/test/exec/exec.go

```diff
@@ -14,14 +14,15 @@ import (
 type Cmd struct {
 	path string
 	args []string
+	env  []string
 }
 
 type Result struct {
 	*icmd.Result
 }
 
-func Command(path string, args ...string) *Cmd {
-	return &Cmd{path: path, args: args}
+func Command(path string, args []string, env []string) *Cmd {
```

Comment:
I would not do this in this PR. It would require you to change all the other calls to this in all the other tests, since they will be broken.

I would maybe follow-up this PR with another PR that adds the 'env' part to `Command` and than fix all the calls in that PR.